src/events/TransactionEvent.php

```diff
@@ -22,4 +22,9 @@ class TransactionEvent extends Event
      * @var Transaction The transaction model
      */
     public $transaction;
+    
+    /**
+     * @var Transaction The transaction model
+     */
+    public $refundTransaction;
```

Comment:
@pdaleramirez should this attribute only be added to the `RefundTransactionEvent` rather than the `TransactionEvent`?